tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -438,6 +474,55 @@ ROOT::Experimental::RClusterDescriptorBuilder::CommitColumnRange(DescriptorId_t
    return RResult<void>::Success();
 }
 
+ROOT::Experimental::RClusterDescriptorBuilder &
+ROOT::Experimental::RClusterDescriptorBuilder::AddDeferredColumnRanges(const RNTupleDescriptor &desc)
+{
+   auto traverseSubtree = [&](DescriptorId_t rootFieldId, std::size_t nRepetitionsAtThisLevel, const auto &visitField,
```

Comment:
Comment the lambda. Since this is a nested lambda, perhaps a "simplify me" todo or an extended comment is necessary.